pwndbg/color/theme.py

```diff
@@ -1,11 +1,9 @@
-import pwndbg.config
+import pwndbg.gdblib.config
 
 
-class Parameter(pwndbg.config.Parameter):
-    def __init__(self, name, default, docstring):
-        super(Parameter, self).__init__(name, default, docstring, "theme")
+def add_param(name, default, docstring):
+    return pwndbg.gdblib.config.add_param(name, default, docstring, "theme")
 
 
-class ColoredParameter(Parameter):
-    def __init__(self, name, default, docstring):
-        super(ColoredParameter, self).__init__(name, default, docstring)
+def add_color_param(name, default, docstring):
+    return add_param(name, default, docstring)
```

Comment:
Instead of having `pwndbg.color.theme.add_*` wouldn't it be better to have `pwndbg.gdblib.config.add_theme_param`?

Reply:
I think you're probably right, but I've been avoiding refactoring `pwndbg/color` as long as I can as there are some decisions there I need to think about before doing it. So I can probably revisit this when I get around to that refactor.